anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -118,12 +118,24 @@ def pairwise_ellip_expansion(
 
                     rot = rotation_matrices[j_global]
                     lengths = ellipsoid_lengths[j_global]
-                    precision, center = radial_basis.compute_gaussian_parameters(
-                        r_ij, lengths, rot
-                    )
-
-                    moments = compute_moments_inefficient_implementation(
-                        precision, center, maxdeg=maxdeg
+                    volume = 4.0 / 3.0 * np.pi * np.product(lengths)
+
+                    length_norm = (
+                        np.product(lengths) * volume * (2.0 * np.pi) ** (3.0 / 2.0)
```

Comment:
As per our GPT friend, we were reminded that the covariance matrix is the product of the squared semiaxes, not just the semiaxes. 

```suggestion
                    length_norm = (
                        np.product(lengths) * np.product(lengths) * (2.0 * np.pi) ** (3.0 / 2.0)
```